docker-compose.yml

```diff
@@ -66,7 +66,33 @@ services:
       - BACKEND_INTERNAL_URL=http://yaptide_flask:6000
       - LOG_LEVEL_ROOT=${LOG_LEVEL_ROOT:-WARNING}
     healthcheck:
-      test: ["CMD-SHELL", "celery --app yaptide.celery.worker inspect ping"]
+      test: ["CMD-SHELL", "celery --app yaptide.celery.simulation_worker inspect ping"]
+    depends_on:
+      redis:
+        condition: service_healthy
+      yaptide_flask:
+        condition: service_healthy
+
+  yaptide_helper_worker:
+    build:
+      context: .
+      dockerfile: Dockerfile-helper-worker
+    image: yaptide_helper_worker
+    container_name: yaptide_helper_worker
+    deploy:
+      resources:
+        limits:
+          # the explanation of below syntax: https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html
+          cpus: ${MAX_CORES-1.0}
+    env_file: .env
+    environment:
+      - CELERY_BROKER_URL=redis://yaptide_redis:6379/0
+      - CELERY_RESULT_BACKEND=redis://yaptide_redis:6379/0
+      - BACKEND_INTERNAL_URL=http://yaptide_flask:6000
+      - LOG_LEVEL_ROOT=${LOG_LEVEL_ROOT:-WARNING}
+      - FLASK_SQLALCHEMY_DATABASE_URI=postgresql+psycopg://${POSTGRES_USER:-yaptide_user}:${POSTGRES_PASSWORD:-yaptide_password}@postgresql:5432/${POSTGRES_DB:-yaptide_db}
```

Comment:
are all of those needed here ? do we call back backend via `BACKEND_INTERNAL_URL` ?

Reply:
Yes, backend url is used.